### PR TITLE
provider/aws: Add support for EC2 Placement Group

### PR DESCRIPTION
**Pull request by radeksimko**
### Test plan

```
$ make testacc TEST=./builtin/providers/aws TESTARGS='-run=PlacementGroup' 2>~/tf.log
```
```
go generate ./...
TF_ACC=1 go test ./builtin/providers/aws -v -run=PlacementGroup -timeout 90m
=== RUN   TestAccAWSPlacementGroup_basic
--- PASS: TestAccAWSPlacementGroup_basic (6.99s)
PASS
ok  	github.com/hashicorp/terraform/builtin/providers/aws	7.012s
```

### CODE REVIEW

File: builtin/providers/aws/resource_aws_placement_group.go [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+		Read:   resourceAwsPlacementGroupRead,
+		Delete: resourceAwsPlacementGroupDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"strategy": &schema.Schema{
```
Comment (stack72) [inline builtin/providers/aws/resource_aws_placement_group.go]: I think we should set a default value of `Cluster` here as it is the **only** supported value atm

Comment (stack72) [inline builtin/providers/aws/resource_aws_placement_group.go]: Ah but we need to have `ForceNew` enabled if there is no update :(

**Comment by stack72** [comment]
This looks good. The only issue I would have with it is that we allow any value as the `strategy` and then wait for AWS to validate the input but that's just $0.02 :)

File: builtin/providers/aws/resource_aws_placement_group.go (hunk 1 above)
Comment (radeksimko) [inline builtin/providers/aws/resource_aws_placement_group.go]: I'm not quite convinced it's a good idea to make it default, but I don't know Amazon's plans for the future.

**Comment by stack72** [comment]
I've tested this and it looks good - i guess this can be merged pretty easily :)

**Comment by stack72** [comment]
@phinze FYI, I checked this branch out and was able to manage placement groups with it. So I can very this is good

**Comment by phinze** [comment]
Looking good! Nice work as usual @radeksimko :grinning:

**Pull request merged**